C#/Benchmark.cs

```diff
@@ -16,10 +16,8 @@ public static class Benchmark
         private static Complex s_one = Complex.One;
 
         // Public function
-        public unsafe static Complex[] Fft(Complex[] xy_in, Complex[] xy_out)
+        public static void Fft(Complex[] xy_in, Complex[] xy_out)
         {
-            PhasevecInit();
```

Comment:
Why to check every time if the vector has been initialized.
Jump + null check + return takes time.